libs/formAnswers.js

```diff
@@ -103,24 +103,31 @@ function getMultipleFieldAnswers(field, previousAnswers) {
   return answers;
 }
 
-function getInitialValue(field, user, previousAnswers) {
+function getInitialValue(field, users, previousAnswers) {
   let initialValue;
 
   field.loadPrevious.forEach(matchString => {
+    let user;
     const strArray = matchString.split('.');
-    if (strArray[0] === 'user') {
-      initialValue = getUserInfo(user, strArray.slice(1)) || initialValue;
-      if (initialValue) {
-        return formatAnswer(field.id, field.tags, initialValue);
-      }
+    switch (strArray[0]) {
+      case 'user':
+        user = users.find(element => element.role === 'applicant');
+        initialValue = getUserInfo(user, strArray.slice(1)) || initialValue;
+        break;
+      case 'coApplicant':
+        user = users.find(element => element.role === 'coapplicant');
+        initialValue = getUserInfo(user, strArray.slice(1)) || initialValue;
+        break;
```

Comment:
Somewhat redundant code. You could do something along the lines of `users.find(user => [ 'applicant', 'coapplicant' ].includes(user.role))`. Then you could also write a if statement instead of a switch case.

```JS
const formUserTypes = ['user', 'coApplicant']
if (formUserTypes.includes(strArray[0])){
  const userCaseRoles = ['applicant', 'coapplicant']
  user = users.find(user => userCaseRole.includes(user.role))
  initialValue = getUserInfo(...)
} else {
  ...
}
```